crates/sui-framework/sources/governance/validator_set.move

```diff
@@ -46,6 +48,9 @@ module sui::validator_set {
         /// Every time a change request is received, this set is updated.
         /// TODO: This is currently not used. We may use it latter for enforcing min/max stake.
         next_epoch_validators: vector<ValidatorMetadata>,
+
+        /// Table from the ID of the StakedSui object to the validator address.
+        delegation_withdraw_queue: LinkedTable<ID, address>,
```

Comment:
Should this be the validator address or the object ID (pending the address -> ID switch)?